lesson07/assignment/my-app-name/src/views/Images.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+<!--Initially I thought about using the bootstrap-vue implementation of carousel (https://bootstrap-vue.org/docs/components/carousel),
+but my original code worked fine so I just refactored it a little bit -->
+  <div id="carousel-container" class="d-flex justify-content-center mt-2">
+    <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
+      <ol class="carousel-indicators">
+        <li data-target="#carouselExampleControls" data-slide-to="0" class="active"></li>
+        <li data-target="#carouselExampleControls" data-slide-to="1"></li>
+        <li data-target="#carouselExampleControls" data-slide-to="2"></li>
+        <li data-target="#carouselExampleControls" data-slide-to="3"></li>
+        <li data-target="#carouselExampleControls" data-slide-to="4"></li>
+      </ol>
+      <!-- Refactored part of original code where I am dynamically making the carousel images -->
+      <div class="carousel-inner">
+        <div v-for="(image, index) in images" class="carousel-item" :class=" { active: index==0 }" v-bind:key="image" >
```

Comment:
Great work with your v-for loop. You'll want to make sure your keys are either strings or integers though. This causes the errors in the web console. You can fix this by using image.alt or some other unique int/string for your key